src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const transformToNumber = (str) => {
+  return parseFloat(str.replace(/[$,]/g, ''));
+};
+const getEmployees = (list) => {
+  return list.map((employee) => {
+    return {
+      name: employee.textContent.trim(),
+      salary: transformToNumber(employee.getAttribute('data-salary')),
+    };
+  });
+};
+const SortList = (list) => {
+  return list.sort((a, b) => b.salary - a.salary);
+};
+
+const employeesArray = Array.from(document.querySelectorAll('ul li'));
+const employeesData = employeesArray.map((employee) => {
+  getEmployees(employee);
```

Comment:
There is an issue here. The `getEmployees` function is expected to return a transformed list of employees, but you are not capturing its return value. You should assign the result of `getEmployees(employee)` to a variable or directly use it in the `employeesData` array.